frontend/server/app.ts

```diff
@@ -218,9 +228,9 @@ function createUIServer(options: UIConfigs) {
     app.use,
     [
       // Original API endpoint is /runs/{run_id}:reportMetrics, but ':reportMetrics' means a url parameter, so we don't use : here.
-      `/${apiVersionPrefix}/runs/*reportMetrics`,
-      `/${apiVersionPrefix}/workflows`,
-      `/${apiVersionPrefix}/scheduledworkflows`,
```

Comment:
Shouldn't we also report 403 when these endpoints are used in v2?

Reply:
Great catch, `reportMetrics` is v1 only. Added both `workflow` and `scheduledworkflows` for v2.